app/src/main/java/eu/quelltext/mundraub/api/MundraubAPI.java

```diff
@@ -152,6 +153,55 @@ protected int loginAsync(String username, String password) {
         return R.string.error_not_specified;
     }
 
+    @Override
+    protected int signupAsync(String email, String username, String password) throws ErrorWithExplanation {
+        try {
+            Map<String, String> formValues = getFormValues(URL_SIGNUP);
+            formValues.put("name", username);
+            formValues.put("mail", email);
+            formValues.put("pass[pass1]", password);
+            formValues.put("pass[pass2]", password);
+            //formValues.put("op", formValues.get("op").replace(" ", "+"));
+            logFormValues("signupAsync", formValues);
+            Thread.sleep(20000); // the signup process uses a honey pod for bots https://www.drupal.org/project/honeypot
```

Comment:
This may be less than 20 seconds. Needs to be tested or read from the Mundraub source code.

Reply:
New issue for this: https://github.com/niccokunzmann/mundraub-android/issues/123